engines/standard_tasks/app/models/standard_tasks/register_decision_task.rb

```diff
@@ -110,6 +112,22 @@ def reject_letter
 
     private
 
+    def create_please_revise_card!
+      return unless revise_decision?
+
+      TaskFactory.build(Task, {
+        title: "Please Revise",
+        role: "user",
+        phase_id: self.phase.id,
+        body: [[{type: 'text', value: revise_letter}]]
+      }).save!
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.